composer.json

```diff
@@ -45,7 +45,7 @@
         "utopia-php/domains": "1.1.*",
 
         "resque/php-resque": "1.3.6",
-        "geoip2/geoip2": "2.10.0",
+        "maxmind-db/reader": "1.8.*",
```

Comment:
Why do we need this dependency when using the native C extension?

Reply:
[As mentioned in their reop](https://github.com/maxmind/MaxMind-DB-Reader-php#optional-php-c-extension), it works along with reader API. so after the composer installs reader in vendor extension can be enabled from vendor/maxmind-db/reader/ext (from their README). is it the wrong approach?

Reply:
I just went through their docs. Your approach seems OK 👍🏻